x/distribution/client/rest/tx.go

```diff
@@ -50,9 +50,49 @@ type (
 		BaseReq         rest.BaseReq   `json:"base_req" yaml:"base_req"`
 		WithdrawAddress sdk.AccAddress `json:"withdraw_address" yaml:"withdraw_address"`
 	}
+
+	// >----------------
+	// setDelegatorWithdrawalAddr is used to generate documentation for setDelegatorWithdrawalAddrHandllerFn
+	setDelegatorWithdrawalAddr struct { // nolint:unused,deadcode
+		Msgs       []types.MsgSetWithdrawAddress `json:"msg"`
+		Fee        auth.StdFee                   `json:"fee"`
+		Signatures []auth.StdSignature           `json:"signatures"`
+		Memo       string                        `json:"memo"`
+	}
+
+	// withdrawDelegatorReward is used to generate documentation for withdrawDelegatorRewardHandlerFn
+	withdrawDelegatorReward struct { // nolint:unused,deadcode
+		Msgs       []types.MsgWithdrawDelegatorReward `json:"msg"`
+		Fee        auth.StdFee                        `json:"fee"`
+		Signatures []auth.StdSignature                `json:"signatures"`
+		Memo       string                             `json:"memo"`
+	}
+
+	// withdrawValidatorRewards is used to generate documentation for withdrawValidatorRewardsHandllerFn
+	withdrawValidatorRewards struct { // nolint:unused,deadcode
+		Msgs       []types.MsgWithdrawValidatorCommission `json:"msg"`
+		Fee        auth.StdFee                            `json:"fee"`
+		Signatures []auth.StdSignature                    `json:"signatures"`
+		Memo       string                                 `json:"memo"`
+	}
 )
 
-// Withdraw delegator rewards
+// withdrawDelegatorRewardsHandlerFn implements a withdraw rewards handler that is responsible
+// for constructing a properly formatted delegator withdraw transaction for signing
+//
+// @Summary Generate a withdraw delegator rewards transaction
+// @Description Generate a withdraw delegator rewards transaction that is ready for signing
+// @Tags distribution
+// @Accept json
+// @Produce json
+// @Param delegatorAddr path string true "delegator address"
+// @Param body body rest.withdrawRewardsReq true "The data required to withdraw rewards"
```

Comment:
why are there two `body` here?

Reply:
param name,param type,data type,is mandatory?,comment attribute(optional)